modules/simulation/DCSupplySimulator/main/power_supply_DCImpl.hpp

```diff
@@ -61,15 +61,15 @@ class power_supply_DCImpl : public power_supply_DCImplBase {
 
     // ev@3370e4dd-95f4-47a9-aaec-ea76f34a66c9:v1
     // insert your private definitions here
-    double settings_connector_export_voltage;
-    double settings_connector_import_voltage;
-    double settings_connector_max_export_current;
-    double settings_connector_max_import_current;
+    double settings_connector_export_voltage{0.0};
+    double settings_connector_import_voltage{0.0};
+    double settings_connector_max_export_current{0.0};
+    double settings_connector_max_import_current{0.0};
     types::power_supply_DC::Mode mode;
```

Comment:
This should probably also be initialized with
`types::power_supply_DC::Mode::Off`

Reply:
Nice catch! I will add this, too :)